modules/database/audit-autoloaded-options/hooks.php

```diff
@@ -26,3 +26,101 @@ function perflab_aao_add_autoloaded_options_test( $tests ) {
 	return $tests;
 }
 add_filter( 'site_status_tests', 'perflab_aao_add_autoloaded_options_test' );
+
+/**
+ * Register custom REST API route for updating autoload.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_register_rest_route() {
+	register_rest_route(
+		'perflab-aao/v1',
+		'/update-autoload/(?P<option_name>[a-zA-Z0-9_-]+)',
+		array(
+			'methods'  => 'POST',
+			'callback' => 'perflab_aao_update_autoload_rest',
+		)
+	);
+}
+add_action( 'rest_api_init', 'perflab_aao_register_rest_route' );
+
+/**
+ * Callback for updating autoload via REST API.
+ *
+ * @since n.e.x.t
+ *
+ * @param WP_REST_Request $request Request object.
+ * @return WP_REST_Response
+ */
+function perflab_aao_update_autoload_rest( $request ) {
+	$option_name = $request->get_param( 'option_name' );
+	$autoload    = $request->get_param( 'autoload' );
+	$value       = $request->get_param( 'value' );
+
+	if ( empty( $option_name ) || empty( $autoload ) ) {
+		return new WP_REST_Response(
+			array(
+				'success' => false,
+				'message' => __( 'Invalid option name or autoload value.', 'performance-lab' ),
+			),
+			400
+		);
+	}
+
```

Comment:
I think this could be missing some validation here. Shouldn't it first be checking to see if the option exists? If not, then it should return an error. Maybe this is overkill as the validation is handled by `wp_set_option_autoload()`.